.gitignore

```diff
@@ -28,3 +28,5 @@ captures/
 # Fleet
 .fleet/
 .docusaurus/
+android/quest/src/main/assets/configs/chss/
```

Comment:
Is it possible to make this generic to ignore any subdirectory inside `android/quest/src/main/assets/configs` other an `app`? This will be so helpful during development.